src/al-sap/al_service_data_unit.cpp

```diff
@@ -39,6 +40,13 @@ void AlServiceDataUnit::appendToPayload(const unsigned char* data, size_t length
 std::vector<unsigned char> AlServiceDataUnit::serialize() const {
     std::vector<unsigned char> serializedData;
 
+    // Calculate SDU size
+    uint32_t fragments_size = 3;// is fragment + is last fragment + fragment id
```

Comment:
I'd personally prefer this since it removes the "magic number-ness" out of it even though it is commented, using the code in combination with the comment to make it clear that each of those elements is a byte and that's why it is `3`
```suggestion
    // is fragment + is last fragment + fragment id
    uint32_t fragments_size = sizeof(uint8_t) + sizeof(uint8_t) + sizeof(uint8_t);
```

Reply:
Addressed in the latest commit